spice_simulation_generator/component_sim.py

```diff
@@ -0,0 +1,33 @@
+# Creates spice deck to simulate a particular component
```

Comment:
Perhaps add a comment here that this is an example of using the spice_gen.py module.